cmd/logcli/tail.go

```diff
@@ -59,8 +57,12 @@ func tailQuery(out output.LogOutput) {
 			}
 
 			for _, entry := range stream.Entries {
-				lbls := mustParseLabels(labels)
+				var lbls label.Labels
+				if !*noLabels {
+					lbls = mustParseLabels(labels)
```

Comment:
in case `*noLabels` true `labels` will be empty string. So, `mustParseLabels(labels)` will throw error.